packages/app/src/utils/formatDate.ts

```diff
@@ -165,19 +161,53 @@ function getFormattedDate(date: Date, style: formatStyle) {
     case 'weekday-medium':
       return WeekdayMedium.format(date);
 
-    case 'relative':
-      return typeof window === 'undefined'
-        ? DayMonth.format(date)
-        : isToday(date)
-        ? siteText.utils.date_today
-        : isYesterday(date)
-        ? siteText.utils.date_yesterday
-        : isDayBeforeYesterday(date)
-        ? siteText.utils.date_day_before_yesterday
-        : DayMonth.format(date);
-
     case 'day-month':
     default:
       return DayMonth.format(date);
   }
 }
+
+/**
+ * Returns one of `vandaag` | `gisteren` | `eergisteren`.
+ * If given date is more than 2 days ago it will return `undefined`.
+ */
+export function formatRelativeDate(input: DateInput) {
+  const date = getDate(input);
+
+  if (typeof window === 'undefined') {
+    throw new Error('formatRelativeDate cannot be called server-side');
+  }
```

Comment:
```suggestion
  assert (window, 'formatRelativeDate cannot be called server-side') 
```

I think Next might have a function or variable to tell you if you're server-side, so maybe that's more elegant than checking window

Reply:
nope: https://github.com/vercel/next.js/pull/7651